todo_app/app.py

```diff
@@ -1,15 +1,34 @@
 from flask import Flask
+from flask import redirect, render_template, url_for
+from flask import request
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import *
```

Comment:
Generally wildcard imports are discouraged ([even by the official Python docs](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package)) - it can easily lead to name clashes & odd behaviour. Here importing the relevant functions directly would probably be cleaner, e.g. `from todo_app.data.session_items import get_items, add_item, save_item`